Modelica_LinearSystems2/Controller/Internal/DiscreteStateSpace.mo

```diff
@@ -129,7 +129,7 @@ initial equation
   elseif init == Types.Init.InitialOutput and nx>0 then
 //  y=y_start;
 //  xd[ny+1:nx]=[zeros(nx-ny,ny),identity(nx-ny)]*Modelica.Math.Matrices.solve(identity(nx)-discreteSystem.A,discreteSystem.B*u);
-      xd = Modelica.Math.Matrices.equalityLeastSquares(identity(nx)-discreteSystem.A, -discreteSystem.B*u, discreteSystem.C, y_start - discreteSystem.D*u);
+      xd = Matrices.equalityLeastSquares(identity(nx)-discreteSystem.A, -discreteSystem.B*u, discreteSystem.C, y_start - discreteSystem.D*u);
 

```

Comment:
Are you sure you want to remove absolute qualifiers like this when you are accessing classes from outside `Modelica_LinearSystems2`. There lead to dangerous wrong lookups.

Reply:
There is "import Modelica.Math.Matrices;" in that class. Would this be a case even if such an import exists?

Reply:
Ah ok in that case it's probably fine.

Reply:
Phew, I'm relieved! Anyway, thanks for your review!